unit/models/card.py

```diff
@@ -366,10 +450,22 @@ def to_json_api(self) -> Dict:
         if self.tags:
             payload["data"]["attributes"]["tags"] = self.tags
 
+        if self.limits:
+            payload["data"]["attributes"]["limits"] = self.limits
+
         return payload
 
     def __repr__(self):
-        json.dumps(self.to_json_api())
+        return json.dumps(self.to_json_api())
+
+
+class PatchBusinessDebitCard(PatchBusinessCard):
+    pass
+
+
+class PatchBusinessCreditCard(PatchBusinessCard):
+    def to_json_api(self) -> Dict:
+        return super().to_json_api("businessCreditCard")
```

Comment:
I know its in their cards branch, but is `businessCreditCard` a real type? I don't see it in the docs?

Reply:
It's in the Charge card docs: https://guides.unit.co/business-charge-cards/#technical-implementation